layouts/page/cudf-polars.html

```diff
@@ -0,0 +1,441 @@
+  <div class="col">
+    <div class="container">
+      <h1 class="mb-4">Accelerate Polars with RAPIDS </h1>
+      <div class="row">
+        <div class="col py-3">
+          <h2><i class="fa-light fa-cloud-arrow-down"></i> Try Now</h2>
+          <p>Try cuDF Polars on <a href="https://colab.research.google.com/github/rapidsai-community/showcase/blob/main/accelerated_data_processing_examples/polars_gpu_engine_demo.ipynb" target="_blank">Google CoLab Right Now <i class="fa-solid fa-arrow-up-right"></i> </a></p>
+        </div>
+        <div class="col py-3">
+          <h2><i class="fa-light fa-chalkboard-user"></i> Learn More</h2>
+          <p>Find out more on the <a href="https://docs.pola.rs/user-guide/gpu-support/" target="_blank">Polars GPU Support Page <i class="fa-solid fa-arrow-up-right"></i> </a></p>
+        </div>
+        <div class="col py-3">
+          <h2><i class="fa-light fa-code-merge"></i> Share Feedback</h2>
+          <p>Share feedback on the <a href="https://github.com/pola-rs/polars/issues" target="_blank">Polars GitHub Page <i class="fa-solid fa-arrow-up-right"></i> </a></p>
+        </div>
+      </div>
+    </div>
+  </div>
+  </div>
+</section>
+
+
+<!-- chart.js -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/4.4.1/chart.umd.min.js"
+  integrity="sha512-CQBWl4fJHWbryGE+Pc7UAxWMUMNMWzWxF4SQo9CgkJIN1kx6djDQZjh3Y8SZ1d+6I+1zze6Z7kHXO7q3UyZAWw=="
+  crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+
+<script>
+  const ctx1 = document.getElementById('perfchartdata');
+  new Chart(ctx1, {
+    type: 'bar',
+    data: {
+      labels: ['20GB', '40GB', '60GB'],
+      datasets: [
+        {
+          label: '.collect()',
+          barPercentage: 0.6,
+          data: [35,67,136], 
+          backgroundColor: 'rgb(33, 150, 243, 0.8)'
+        },
+        {
+          label: '.collect(engine="gpu")',
+          barPercentage: 0.6,
+          data: [12,19,32],
+          backgroundColor: 'rgb(115, 6, 255, 0.8)'
+        }]
+    },
+    options: {
+      indexAxis: 'x',
+      grouped: true,
+      scales: {
+        y: {
+          type: 'linear',
+          beginAtZero: true,
+          grid: {
+            display: true,
+            borderWidth: 0
+          },
+          title: {
+            display: true,
+            text: 'Time (s)'
+          }
+        },
+        x: {
+          ticks: {
+            font: {
+              weight: 'bold'
+            }
+          },
+          grid: {
+            display: false,
+            drawTicks: false,
+            borderWidth: 3
+          },
+          title: {
+            display: true,
+            text: 'Dataset Size (GB)'
+          }
+        }
+      },
+      plugins: {
+        legend: {
+          display: true
+        },
+        title: {
+          display: true,
+          text: 'PDS-H benchmark 80GB query processing time across a range of dataset sizes'
```

Comment:
Can we say - 

'Query Processing time across a range of dataset sizes' 

This is not just for 80 GB